packages/assets-controllers/src/token-prices-service/codefi-v2.test.ts

```diff
@@ -74,13 +74,28 @@ describe('CodefiTokenPricesServiceV2', () => {
           },
         });
 
-      await expect(
-        new CodefiTokenPricesServiceV2().fetchTokenPrices({
-          chainId: '0x1',
-          tokenAddresses: ['0xAAA', '0xBBB', '0xCCC'],
+      const result = await new CodefiTokenPricesServiceV2().fetchTokenPrices({
+        chainId: '0x1',
+        tokenAddresses: ['0xAAA', '0xBBB', '0xCCC'],
+        currency: 'ETH',
+      });
+      expect(result).toStrictEqual({
+        '0xAAA': {
+          tokenAddress: '0xAAA',
+          value: undefined,
           currency: 'ETH',
-        }),
-      ).rejects.toThrow('Could not find price for "0xAAA" in "ETH"');
+        },
+        '0xBBB': {
+          tokenAddress: '0xBBB',
+          value: 33689.98134554716,
+          currency: 'ETH',
+        },
+        '0xCCC': {
+          tokenAddress: '0xCCC',
+          value: 148.1344197578456,
+          currency: 'ETH',
+        },
+      });
     });
 
     it('throws if the currency cannot be found in the response data', async () => {
```

Comment:
Similar here — do we want to update the description to reflect what should happen instead of throwing or not throwing?